lib/rand/dune

```diff
@@ -16,7 +16,7 @@
    (run mkdir -p .cargo)
    (run cp cargo-config.toml .cargo/config.toml)
    ; cargo build
-   (run cargo build --release --offline)
+   (run cargo build --release --offline --locked)
```

Comment:
interesting. I've think I've seen also `(locks Cargo.lock)` in dune files to do the same thing? Not sure.

Thanks for the PR!